extension/filterProductProperties.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function (context, input, cb) {
+  const propertiesToRemove = ['Apply Map']
+  const propertiesToReturen = input.properties.filter(function (property) {
```

Comment:
There is a typo in `propertiesToReturen`

Reply:
corrected name